jupyter_server/serverapp.py

```diff
@@ -1606,6 +1606,33 @@ def init_httpserver(self):
                               'no available port could be found.'))
             self.exit(1)
 
+    def _init_asyncio_patch(self):
```

Comment:
I think we can make this a `staticmethod` and call this in conftest—thereby removing the copy in conftest. I'll make these additions myself